uwblueprint/childrens-aid-society
# Add migration to update intake and caregiver enums

## Notion ticket link




## Implementation description
* Create migration to update the intake/caregiver enums changed in #77 



## Steps to test
1. Start up the containers: `docker compose up`
2. Upgrade your database: `docker exec -it cas_py_backend /bin/bash -c "flask db upgrade"`
3. Exec into the database and check the enum types: `docker exec -it cas_db /bin/bash -c "psql -U postgres -d cas"` and `SELECT UNNEST(enum_range(null::intakes_court_status))  AS intakes_court_status;`
4. Repeat 3 for the other enums (`intakes_first_nation_heritage` and `caregivers_relationship_to_child`)
5. Try downgrading your database and upgrading just to make sure it works: `docker exec -it cas_py_backend /bin/bash -c "flask db downgrade && flask db upgrade"`




## What should reviewers focus on?
* do the new enum types match?


## Checklist
